crates/transcode/src/env_types.rs

```diff
@@ -45,15 +45,15 @@ use std::{
 /// Provides custom encoding and decoding for predefined environment types.
 #[derive(Default)]
 pub struct EnvTypesTranscoder {
-    encoders: HashMap<u32, Box<dyn CustomTypeEncoder>>,
-    decoders: HashMap<u32, Box<dyn CustomTypeDecoder>>,
+    encoders: HashMap<u32, Box<dyn CustomTypeEncoder + Send + Sync>>,
```

Comment:
Would it also work if we added those constraints to the trait itself e.g. `CustomTyoeEncoder: Send + Sync`?

Reply:
Yeah, that works as well, updated the PR.